tests/TestCase/Database/Schema/SqliteSchemaTest.php

```diff
@@ -384,6 +392,35 @@ public function testDescribeTable()
         }
     }
 
+    /**
+     * Tests SQLite views
+     *
+     * @return void
+     */
+    public function testDescribeView()
```

Comment:
```suggestion
    public function testDescribeViewWithComputedValues()
```

I think this captures a bit more of the scenario this test is covering.

Reply:
My idea was to leave that test open for other assertions that may be needed with views. I am okay altering the name as you suggested if you don't think that's pertinent.

Reply:
I'm not too fussy about the names. :smile: